package.json

```diff
@@ -149,6 +148,7 @@
     "invariant": "^2.2.2",
     "lodash.flowright": "^3.5.0",
     "lodash.isequal": "^4.5.0",
-    "prop-types": "^15.6.0"
+    "prop-types": "^15.6.0",
+    "react-dom": "16.6.1"
```

Comment:
Note that tests only pass if this is an exact version. If I relax the constraint with `^` or explicitly update to the latest version (16.6.3), tests begin to fail.